pennylane/labs/tests/resource_estimation/ops/test_identity.py

```diff
@@ -114,7 +114,7 @@ def test_resource_rep(self):
     def test_resource_params(self):
         """Test the resource params are correct"""
         op = re.ResourceGlobalPhase(0.1, wires=0)
-        assert op.resource_params() == {}
+        assert op.resource_params == {}
```

Comment:
codefactor complains here.

Reply:
That's weird, it didn't complain about the same line in other tests though. It's probably just a fluke.

Reply:
I agree, this seems like some bug with code factor. 👍🏼